happi/tests/test_client.py

```diff
@@ -21,7 +22,7 @@ def xdg_config_home(tmp_path):
 
 
 @pytest.fixture(scope='function')
-def happi_cfg(tmp_path, xdg_config_home):
+def happi_cfg(xdg_config_home):
     # Store current happi config
     xdg_cfg = os.environ.get("XDG_CONFIG_HOME", '')
     happi_cfg = os.environ.get("HAPPI_CFG", '')
```

Comment:
Wait, hold on - I know you didn't add this, but the test suite overwrites `xdg_config_home/happi.cfg`?! I think it's time to fix this...

(Note: you need to expand the diff here)

Reply:
It looks like the fixture makes a new new `xdg_config_home` directory in a temp path?  This seemed ok to me, since there are tests that cover `client.find_config` looking at the `XDG_CONFIG_HOME` directory.